### PR TITLE
New branch

### PR DESCRIPTION
**Pull request by DagiPaulos**

### CODE REVIEW

**Review by Clemens-Bretscher — COMMENTED** [review]
lgtm

File: README.md [hunk 1]
```diff
@@ -4,5 +4,7 @@ jari jari jara
```
Comment (DagiPaulos) [inline README.md:4]: a new msg is passed to the project manager

File: README.md [hunk 2]
```diff
@@ -4,5 +4,7 @@ jari jari jara
 
 i creared some changes
 
+i creared some changes
+
 Clement and Paulos are working on this project right now
```
Comment (DagiPaulos) [inline README.md:9]: Tis the second msg that I have passed to the project manager

File: textfile.txt [hunk 3]
```diff
@@ -0,0 +1,2 @@
+Hello world
```
Comment (DagiPaulos) [inline textfile.txt:1]: Make your codes as concise as possible

**Review by DagiPaulos — COMMENTED** [review]
Great Job!!

**Review by DagiPaulos — COMMENTED** [review]
Good Job!!!

**Pull request merged**